### ISSUE TITLE
Semantics of routes

### ISSUE THREAD

**Issue by McSim182** [ModPhoenix/incode_server#1]
https://github.com/ModPhoenix/incode_server/blob/master/routes/task.route.js

```
router.get('/', task_controller.tasks_all);
router.post('/create', task_controller.task_create);
router.get('/:id', task_controller.task_details);
router.put('/:id/update', task_controller.task_update);
router.delete('/:id/delete', task_controller.task_delete);
```

this should be refactored according to REST standarts
```
get('/', getAll)
get('/:id', getById)
post('/', create)
put('/:id', update)
delete('/:id', delete)
```

**Comment by ModPhoenix** [comment]
Fixed in #2

**Comment by McSim182** [comment]
Names of methods could stay the same, idea was to remove 'create', 'delete', 'get', 'update'  from routes
